tests/network/vmi_slirp_interface.go

```diff
@@ -163,8 +163,7 @@ var _ = SIGDescribe("Slirp Networking", func() {
 			vmi := *vmiRef
 			vmi, err = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(vmi)
 			Expect(err).ToNot(HaveOccurred())
-			tests.WaitForSuccessfulVMIStartIgnoreWarnings(vmi)
-			tests.GenerateHelloWorldServer(vmi, 80, "tcp", console.LoginToCirros, true)
+			vmi = tests.WaitUntilVMIReady(vmi, console.LoginToCirros)
```

Comment:
This has an implicit logic change from ignoring warnings to not ignoring warning.

Reply:
Yeah. It works fine without it, and half of this test was a crude copy-paste. But since I cannot track the original motivation, and I'm not in a mood to toy with occult powers, I'm removing this change.